keyboards/7c8/framework/config.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 Steven Nguyen
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "config_common.h"
+
+#define VENDOR_ID 0x07c8
```

Comment:
This vendor ID is in use, and should be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x07c8&p=&n=